analyzer/uncategorized/runtimes.go

```diff
@@ -15,7 +15,7 @@ func OpenUtxNoVerify(utx *sdkTypes.UnverifiedTransaction) (*sdkTypes.Transaction
 	if len(utx.AuthProofs) == 1 && utx.AuthProofs[0].Module != "" {
 		switch utx.AuthProofs[0].Module {
 		case "evm.ethereum.v0":
-			tx, err := decodeEthRawTx(utx.Body, 42262)
+			tx, err := decodeEthRawTx(utx.Body, 23294)
```

Comment:
need to add a few layers of parameters. might make sense to wire this in all the way from the config

Reply:
This is completely separate from [RuntimeInfo.ID](https://github.com/oasisprotocol/oasis-sdk/blob/5e0353622f049fb53c52b3e7c8e54aa2d998bbdc/client-sdk/go/config/default.go#L90-L90)? I'm surprised we don't have this chainID anywhere in the Go codebase, only Rust.

Also, nit, prefer the hex version here because that's how it's used everywhere else in oasisprotocol repos.

The term chainID is already used in the indexer and has a fixed value of `oasis_3`, btw; perhaps we can get rid of it? I'm not sure when it would change.

Reply:
:+1: on hex

yes on this being separate from runtime ID. runtime ID is an oasis concept. chain ID in this usage is a concept from ethereum-compatible systems

Reply:
getting rid of the check